sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java

```diff
@@ -1848,4 +1847,19 @@ private static class JurisdictionValues {
 		protected String facilityDetails;
 	}
 
+	public boolean hasPointOfEntry(CaseDataDto caze) {
```

Comment:
Since the aim of this method is only to check whether the case has a point of entry or not, please change the backend method to return a boolean instead of the whole point of entry; don't query for the point of entry, but only check whether the field is not null.